miso-web/src/main/webapp/scripts/form_container.js

```diff
@@ -40,6 +36,10 @@ FormTarget.container = (function() {
           getDisplayValue: function(container) {
             return container.model.alias;
           }
+        }, {
+          title: 'Change Model',
+          type: 'special',
```

Comment:
(Just leaving a note here that we spoke earlier about converting the container model field to a dropdown instead.)

Reply:
Changed. Dropdown will sometimes contain a mix of models from multiple instruments when the container isn't connected to any runs, but it should never be too long of a list